test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js

```diff
@@ -47,14 +50,11 @@ define([
                 description:
                     'Valid file extensions for FASTA: .fasta, .fna, .fa   Valid file extensions for FASTQ: .fastq, .fnq, .fq; Compressed files (containing files with vaild extentions): .zip, .gz, .bz2, .tar.gz, .tar.bz2',
                 disabled: 0,
-                dynamic_dropdown_options: {
-                    data_source: 'ftp_staging',
-                    multiselection: 0,
-                    query_on_empty_input: 1,
-                    result_array_index: 0,
-                    service_params: null,
+                text_options: {
```

Comment:
Turn this into testing a simple text input widget. It should be focused on the input widget wrapper component, anyway, and we don't want to have to deal with the dynamic dropdown stuff.

Reply:
A wise choice!